packages/client/hmi-client/src/services/models/funman-service.ts

```diff
@@ -60,6 +60,17 @@ export async function makeQueries(body: FunmanPostQueriesRequest, modelId: strin
 	}
 }
 
+export async function cancelQueries(taskId: string) {
+	try {
+		const resp = await API.delete(`/funman/queries/${taskId}`);
+		const output = resp.data;
+		return output;
```

Comment:
This is how we've been doing in most service calls
```suggestion
		const { data } = await API.delete(`/funman/queries/${taskId}`);
		return data;
```